pkg/utils/options/options.go

```diff
@@ -41,20 +42,24 @@ func MustParse() Options {
 
 // Options for running this binary
 type Options struct {
-	ClusterName     string
-	ClusterEndpoint string
-	MetricsPort     int
-	HealthProbePort int
-	WebhookPort     int
-	KubeClientQPS   int
-	KubeClientBurst int
+	ClusterName           string
+	ClusterEndpoint       string
+	MetricsPort           int
+	HealthProbePort       int
+	WebhookPort           int
+	KubeClientQPS         int
+	KubeClientBurst       int
+	AWSNodeNameConvention string
 }
 
 func (o Options) Validate() (err error) {
 	err = multierr.Append(err, o.validateEndpoint())
 	if o.ClusterName == "" {
 		err = multierr.Append(err, fmt.Errorf("CLUSTER_NAME is required"))
 	}
+	if o.AWSNodeNameConvention != "ip-name" && o.AWSNodeNameConvention != "resource-name" {
```

Comment:
Can we default to ip-name, for now? I'd hate for folks to have to pass this flag in all cases.

Reply:
`ip-name` is the default when parsing the flags. There shouldn't be a need to set this explicitly when running through `main()`. At least that was my intention.